Testing/SystemTests/tests/framework/PolarizationCorrections/HeliumAnalyserEfficiencyTest.py

```diff
@@ -0,0 +1,67 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from abc import ABCMeta, abstractmethod
+from mantid.simpleapi import *
```

Comment:
In several of these files we seem to import all of the `simpleapi`. I realise this may have just been copied across from other tests, but would it be worth just importing the algorithms that we're using?